doc/indices.sgm

```diff
@@ -8,15 +8,29 @@
         </title>
         <para>
           <application>pgSphere</application> uses <literal>GiST</literal>
-          to create spherical indices. An index speeds up the execution
-          time of operators <link
+          and Block Range INdexing (<literal>BRIN</literal>) algorithms to create
+          spherical indices.
+          <literal>GiST</literal> indexes utilize an R-tree implementation for
+          spherical objects, while <literal>BRIN</literal> indexes are based on "summarization"
+          of data blocks (<literal>pages</literal>) on physical storage in order to
+          organize data searches on ranges of summarized data that can be easily skipped
+          on the base of search filters (see <ulink
+          url="https://www.postgresql.org/docs/current/brin-intro.html"><citetitle>
+          PostgreSQL documentation</citetitle></ulink> for further details on BRIN indexes).
+          As a consequence, BRIN indexes result to be really small indexes (up to 1000 times smaller
```

Comment:
Change "result to be really" to "are very".

Reply:
I'm not sure, I understand the final sentence. I would like to change this sencence to something like this: BRIN indexes consume significantly less physical storage than GiST indexes.

Reply:
Fine.

Reply:
@vitcpp it's not only related to less used storage. It's more because indexes are more efficiently used as far as more 8kB pages of the index are in-memory kept: the more RAM you have available to host the shared buffers in PostgreSQL, the more an index can be used for large amount of data, even if the resulting index is big. The good point of using BRINs is because they fits easily in the shared buffers even in case of large data stored in relatively small machines.

Reply:
But regarding the correction: it's fine, and it intrinsecally includes what I said above.

Reply:
May be to simplify the sentense like this:
_As a consequence, BRIN indexes **are small** (up to 1000 times smaller ...)?_

I've prepared the commit with such change.

Reply:
> May be to simplify the sentense like this: _As a consequence, BRIN indexes **are small** (up to 1000 times smaller ...)?_
> 
> I've prepared the commit with such change.

No, that's not proper English. Either go with what I wrote in my original comment or what you wrote in the second comment in this thread.

Reply:
Oh, sorry. I've changed as you proposed. Thank you.